pyrex.py

```diff
@@ -165,18 +165,27 @@ def capture(args):
         builddir = os.environ['BUILDDIR']
         conffile = os.path.abspath(os.path.join(builddir, 'conf', 'pyrex.ini'))
         oeinit = os.environ['PYREX_OEINIT']
+        force_conffile = os.environ.get('PYREXCONFFILE', '')
 
         user_config = Config()
 
-        if not os.path.isfile(conffile):
-            copy_templateconf(conffile)
+        if force_conffile:
+            shutil.copyfile(force_conffile, conffile)
```

Comment:
Is this the best way to do it? Can we just remember the path of the file to be used?

Reply:
It seems more straight forward to always copy the file if the variable is set... we *could* remember the path, but I don't see what that would gain us. If you are using this variable, it is pretty much required that you are writing your own init wrapper to set that variable, which means it *should* always be set and remember it gains us nothing.

Reply:
Maybe I'm working from the wrong premise. I'm viewing this $PYREXCONFFILE variable as equivalent to a '-f' option that overrides the default-calculated config file path for a program. You don't expect that the default config file is overwritten as part of the operation of '-f' in those cases.

Reply:
Hmm, ya we can do that